plugin/storage/es/spanstore/schema.go

```diff
@@ -0,0 +1,183 @@
+               "type":"long"
+            },
+            "flags":{
+               "type":"integer"
+            },
+            "logs":{
+               "properties":{
+                  "timestamp":{
+                     "type":"long"
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "process":{
+               "properties":{
+                  "serviceName":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "references":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "refType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "traceID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "spanID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            },
+            "tags":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "key":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "value":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tagType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            }
+         }
+      },
+      "service":{
```

Comment:
@black-adder @badiib thoughts?
it's not difficult to pull them out to separate indices, so this is definitely a viable option. I also don't think we have any searches where we need both within the same index; they're completely decoupled. lmk what you guys think